product_docs/docs/efm/4/04_configuring_efm/01_cluster_properties.mdx

```diff
@@ -576,7 +576,9 @@ To perform maintenance on the primary database when `primary.shutdown.as.failure
 
 <div id="update_physical_slots_period" class="registered_link"></div>
 
-Use the `update.physical.slots.period` property to define the slot advance frequency for database version 12 and later. When `update.physical.slots.period` is set to a positive integer value, the primary agent reads the current `restart_lsn` of the physical replication slots after every `update.physical.slots.period` seconds and sends this information with its `pg_current_wal_lsn` and `primary_slot_name` (if it is set in the postgresql.conf file) to the standbys. The physical slots must already exist on the primary for the agent to find them. If physical slots do not already exist on the standbys, standby agents create the slots and then update `restart_lsn parameter` for these slots. A non-promotable standby doesn't create new slots but updates them if they exist.
+Use the `update.physical.slots.period` property to define the slot advance frequency for database version 12 and later. When `update.physical.slots.period` is set to a positive integer value, the primary agent reads the current `restart_lsn` of the physical replication slots after every `update.physical.slots.period` seconds and sends this information with its `pg_current_wal_lsn` and `primary_slot_name` (if it is set in the postgresql.conf file) to the standbys. The physical slots must already exist on the primary for the agent to find them. If physical slots do not already exist on the standbys, standby agents create the slots and then update `restart_lsn` parameter for these slots. A non-promotable standby doesn't create new slots but updates them if they exist.
```

Comment:
The change in this block is at the end: the back ticks were around "restart_lsn parameter" instead of around "restart_lsn" with "parameter" outside the markup.